### PR TITLE
Add support for multigroup

### LINKED ISSUE #78: Add support for kubebuilder Multi-Version API
# Summary

We should follow the tutorial and add support for multi version APIs.

See https://book.kubebuilder.io/multiversion-tutorial/tutorial.html

### PR DESCRIPTION
**Pull request by iainbowman**
Closes #78

### CODE REVIEW

**Review by maruina — COMMENTED** [review]
Generally looks good, just one question

File: charts/kms-issuer/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.0
```
[the remark this answers is not in the document]
Comment (iainbowman) [reply charts/kms-issuer/Chart.yaml:18]: The chart version has changed as the tests fail if I leave this at the current value, the chart itself has not changed.

[the remark this answers is not in the document]
Comment (iainbowman) [reply charts/kms-issuer/Chart.yaml:18]: https://github.com/Skyscanner/kms-issuer/runs/6687341456?check_suite_focus=true

[the remark this answers is not in the document]
Comment (maruina) [reply charts/kms-issuer/Chart.yaml:18]: Yes, I can see that the CRDs are changing. I got confused by the UI.

**Pull request merged**